diff --git a/Makefile b/Makefile
--- Makefile
+++ Makefile
@@ -75,9 +75,9 @@
 # newer than our target (bin/podman{,-remote}), a rebuild is
 # triggered.
 SOURCES = $(shell find . -path './.*' -prune -o \( \( -name '*.go' -o -name '*.c' \) -a ! -name '*_test.go' \) -print)
 
-BUILDTAGS_CROSS ?= containers_image_openpgp exclude_graphdriver_btrfs exclude_graphdriver_devicemapper exclude_graphdriver_overlay
+BUILDTAGS_CROSS ?= containers_image_openpgp exclude_graphdriver_btrfs btrfs_noversion exclude_graphdriver_devicemapper seccomp apparmor systemd libdm_no_deferred_remove
 CONTAINER_RUNTIME := $(shell command -v podman 2> /dev/null || echo docker)
 OCI_RUNTIME ?= ""
 
 # The 'sort' below is crucial: without it, 'make docs' behaves differently
@@ -141,20 +141,9 @@
 
 GOMD2MAN ?= $(shell command -v go-md2man || echo './test/tools/build/go-md2man')
 
 CROSS_BUILD_TARGETS := \
-	bin/podman.cross.linux.amd64 \
-	bin/podman.cross.linux.ppc64le \
-	bin/podman.cross.linux.arm \
 	bin/podman.cross.linux.arm64 \
-	bin/podman.cross.linux.386 \
-	bin/podman.cross.linux.s390x \
-	bin/podman.cross.linux.mips \
-	bin/podman.cross.linux.mipsle \
-	bin/podman.cross.linux.mips64 \
-	bin/podman.cross.linux.mips64le \
-	bin/podman.cross.freebsd.amd64 \
-	bin/podman.cross.freebsd.arm64
 
 # Dereference variable $(1), return value if non-empty, otherwise raise an error.
 err_if_empty = $(if $(strip $($(1))),$(strip $($(1))),$(error Required variable $(1) value is undefined, whitespace, or empty))
 
@@ -419,9 +408,9 @@
 bin/podman.cross.%:
 	TARGET="$*"; \
 	GOOS="$${TARGET%%.*}"; \
 	GOARCH="$${TARGET##*.}"; \
-	CGO_ENABLED=0 \
+	CGO_ENABLED=1 \
 		$(GO) build \
 		$(BUILDFLAGS) \
 		$(GO_LDFLAGS) '$(LDFLAGS_PODMAN)' \
 		-tags '$(BUILDTAGS_CROSS)' \
