diff --git a/Makefile b/Makefile
index 15d6d9fb6..8b44da6ae 100644
--- a/Makefile
+++ b/Makefile
@@ -65,7 +65,7 @@ SOURCES = $(shell find . -path './.*' -prune -o \( -name '*.go' -a ! -name '*_te

 BUILDFLAGS := -mod=vendor $(BUILDFLAGS)

-BUILDTAGS_CROSS ?= containers_image_openpgp exclude_graphdriver_btrfs exclude_graphdriver_devicemapper exclude_graphdriver_overlay
+BUILDTAGS_CROSS ?= containers_image_openpgp exclude_graphdriver_btrfs exclude_graphdriver_devicemapper exclude_graphdriver_overlay seccomp
 CONTAINER_RUNTIME := $(shell command -v podman 2> /dev/null || echo docker)
 OCI_RUNTIME ?= ""

@@ -133,16 +133,7 @@ export PATH := $(PATH):$(GOBIN)
 GOMD2MAN ?= $(shell command -v go-md2man || echo '$(GOBIN)/go-md2man')

 CROSS_BUILD_TARGETS := \
-	bin/podman.cross.linux.amd64 \
-	bin/podman.cross.linux.ppc64le \
-	bin/podman.cross.linux.arm \
 	bin/podman.cross.linux.arm64 \
-	bin/podman.cross.linux.386 \
-	bin/podman.cross.linux.s390x \
-	bin/podman.cross.linux.mips \
-	bin/podman.cross.linux.mipsle \
-	bin/podman.cross.linux.mips64 \
-	bin/podman.cross.linux.mips64le

 # Dereference variable $(1), return value if non-empty, otherwise raise an error.
 err_if_empty = $(if $(strip $($(1))),$(strip $($(1))),$(error Required variable $(1) value is undefined, whitespace, or empty))
@@ -370,7 +361,7 @@ bin/podman.cross.%: .gopathok
 	TARGET="$*"; \
 	GOOS="$${TARGET%%.*}"; \
 	GOARCH="$${TARGET##*.}"; \
-	CGO_ENABLED=0 \
+	CGO_ENABLED=1 \
 		$(GO) build \
 		$(BUILDFLAGS) \
 		-gcflags '$(GCFLAGS)' \
